ko/admin/admin_utils.rst

```diff
@@ -3279,6 +3280,50 @@ tde
 
     이 옵션 뒤에 오는 값은 **DBA** 의 암호이며 생략하면 프롬프트에서 입력해야 한다.
 
+.. _vacuumdb:
+
+vacuumdb
+--------
+
+ **cubrid vacuumdb** 유틸리티는 삭제된 레코드와 불필요한 mvcc 관련 정보를 데이터베이스의 레코드에서 vacuum하는 데 사용됩니다. 그리고 vacuum에 대한 상태 정보를 얻기 위해서도 사용됩니다. ::
```

Comment:
```suggestion
 **cubrid vacuumdb** 유틸리티는 삭제된 레코드와 불필요한 mvcc 관련 정보를 정리하기 위해 데이터베이스의 레코드에 대하여 vacuum을 수행하거나, vacuum 관련 상태 정보를 얻기 위해 사용된다. **DBA** 사용자만 수행할 수 있다. ::
```

Reply:
There doesn't seem to be any restrictions on being a DBA.